pyproject.toml

```diff
@@ -42,39 +42,34 @@ files = ["dowhy/__init__.py"]
 
 #
 # Dependency compatibility notes:
-# * autogluon requires scikit-learn <1.2.0
 # * numba (imported by econml) requires python <3.11
 #
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 cython = "^0.29.32"
-scipy = "^1.8.1"
-statsmodels = "^0.13.2"
-numpy = "^1.23.1"
-pandas = "^1.4.3"
-networkx = "^2.8.5"
+scipy = ">=1.4.1"
+statsmodels = ">=0.13.2,<1.0"
```

Comment:
`>=0.13.2,<1.0`

isn't this the same as

`^0.13.2` in poetry?

Reply:
I thought so too, but somehow it behaved slightly differently. Might be that it was something local, but just wanted to make sure :)

Reply:
Both should be fine! :)